TESTING.md

```diff
@@ -63,12 +63,47 @@ tests are finished. The command below will give you a shell after all tests are
 
 ```
 # cd keylime-tests
-# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 execute login finish
+# tmt run -vvv prepare discover provision -h virtual -i Fedora-35 -c system execute login finish
 ```
 
 You can use it to inspect test logs or even modify test sources and run your tests
 manually following the method described above. For this purpose you can find test sources under `/var/tmp/tmt/run-*/packit-ci/discover/default/tests/`.
 
+### Running multi-host tests
+
+`tmt` cannot schedule multi-host tests yet. However, we can use `tmt` to do the necessary setup and then execute the test manually.
+
+For `Multihost/basic-attestation` test we need at least 3 test systems that
+will be started using `tmt`.
+
+Let's start with opening 3 terminals on your workstation.
+
+On terminal 1:
+```
+$ git clone https://github.com/RedHat-SP-Security/keylime-tests.git
+```
+
+On all terminals:
+```
+$ cd keylime-tests
+$ tmt run -vvv prepare discover -h fmf -t 'configure_tpm_emulator' -t 'install_upstream_keylime' -t 'Multihost/basic-attestation' provision -h virtual -i Fedora-35 execute login finish
+```
+
+Multihost test won't be run properly but at least tmt will install all test requires and do the setup.
```

Comment:
Do you have a typo here, perhaps, and mean that tmt will install the test requirements?

Reply:
Fixed.